init.sh

```diff
@@ -1,19 +1,16 @@
-# Restore and build the dotnet app
-dotnet restore
-dotnet build
-
-# Install NPM and start the server, it will run side-by-side with the dotnet API server for speed
+# First initial install of npm to ensure it runs npm
 npm install --prefix HoneyDo.Web/ClientApp
-npm start --prefix HoneyDo.Web/ClientApp
+# ./npmStart.sh # runs npm i and then start in a new process
 
 # Setup a postgres docker and migrate the tables to it - if this fails, run the following commands to remove the container and re-create it
 # `docker ps` - copy container ID and replace [container ID] in `docker stop [411b2c4fd3e9]` then `docker rm [411b2c4fd3e9]`
 docker run --name honeydo-db -e POSTGRES_DB=honeydo -e POSTGRES_USER=honeydo-user -e POSTGRES_PASSWORD=mysecretpassword -p 5432:5432 -d postgres
 dotnet ef database update -p ./HoneyDo.Infrastructure/ -s ./HoneyDo.Web/ -c HoneyDoContext
 
 # Install dotnet dev-certs for https and then trust them
-dotnet tool install --global dotnet-dev-certs
-dotnet dev-certs https
+dotnet tool install --global dotnet-dev-certs #will warn if already installed
+dotnet dev-certs https # linux
+dotnet dev-certs https --trust # mac/windows
```

Comment:
I tested this, it doesn't hurt to have both, at least for mac.  Did you want to test this to make sure it's okay to have both on linux?